.github/ISSUE_TEMPLATE/zpa_epic.md

```diff
@@ -0,0 +1,80 @@
+---
+
+name: Zeebe Process Automation team Epic
+about: Prepare a new Epic for the ZPA team
+title: ''
+labels: 'kind/epic'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+
+
+
+## Introduction
+<!-- Provide a short summary about the problem this Epic will solve -->
+
+
+## Concept
+<!-- Provide a high-level explanation of the concept. For example, when a new BPMN element gets added,
+explain how it is supposed to work. You can add relevant pieces of the specification here as well -->
+
+
+## Task Breakdown
+<!-- When splitting the Epic into smaller issues, they should be referenced here. This provides an
+easy overview of all that needs to happen. It also provides a good overview of the progress on the
+Epic thus far. -->
+
+### Broker
+<!-- Tasks that require changes in the Broker -->
+
+### Gateway API
+<!-- Tasks that require changes to the Gateway API. This also includes changes in the clients -->
+
+### Exporters and Metrics
+<!-- Tasks that require changes to the exporters and metrics -->
+
+### Testing
+<!-- Tasks for extending the tests. This does not include unit tests, these should be written as
+part of any other task. Think about things like QA tests and the randomized property tests -->
+
+### Zeebe Process Test
+<!-- Tasks that require changes in Zeebe Process Test. Generally when you make changes in the
+Gateway API, a change needs to be made in Zeebe Process Test as well. -->
+
+### Documentation
+<!-- Tasks that require writing / modifying documentation -->
+
+### Out of scope
+<!-- Tasks that have been thought about, but are out of scope for now -->
+
+## Other Teams
+<!-- The Epic might have an impact on other teams within Camunda. We should create an issue for
+them, explaining the changes they will have to make. -->
+
+- [ ] [Operate](https://github.com/camunda/operate/issues)
+- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
+- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
+- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
+- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)
+
+## Open Questions
+<!-- Any open questions during the preparation of this Epic can be written down here -->
+
+
```

Comment:
🔧 I'd prefer to also add a section for `Decisions` or `Discussions`, where we explain the answers to resolved questions, e.g. specific design decisions, or answers to questions that don't make sense to mention in the rest of the breakdown.

Reply:
I feel like this could be the same category as `Open Question`. Maybe we should rename it to `Discussions and Decisions`. An ongoing discussions/decision is IMO the same as an open question. WDYT?

Reply:
That's a good point, and it might be easier to use because it avoids us having to move things around. 

👿 There's a counter-argument to be made that it's easier to find what is still unclear, or what has been decided in the split variant.

👍 I'm fine with it either way